src/index.c

```diff
@@ -496,12 +504,14 @@ static void II_SortChildren(IntersectIterator *ctx) {
    *    tester list
    */
   IndexIterator **unsortedIts = NULL;
+  IndexIterator **sortedIts = NULL;
 
   for (size_t i = 0; i < ctx->num; ++i) {
     IndexIterator *curit = ctx->its[i];
```

Comment:
btw, shouldn't we bail here instead? if there's an empty iterator in an intersection, it will always be false.

Reply:
Yes, could be an optimization but should be done in another PR.